# Termux/Bionic port: deps/llvm.mk
#
# Point cmake at the Termux zlib, not Julia's empty deps/usr.
--- a/deps/llvm.mk
+++ b/deps/llvm.mk
@@ -94,7 +94,7 @@
 LLVM_CMAKE += -DLLVM_TARGETS_TO_BUILD:STRING="$(LLVM_TARGETS)" -DCMAKE_BUILD_TYPE="$(LLVM_CMAKE_BUILDTYPE)"
 LLVM_CMAKE += -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD:STRING="$(LLVM_EXPERIMENTAL_TARGETS)"
 LLVM_CMAKE += -DLLVM_ENABLE_LIBXML2=OFF -DLLVM_HOST_TRIPLE="$(or $(XC_HOST),$(BUILD_MACHINE))"
-LLVM_CMAKE += -DLLVM_ENABLE_ZLIB=FORCE_ON -DZLIB_ROOT="$(build_prefix)"
+LLVM_CMAKE += -DLLVM_ENABLE_ZLIB=FORCE_ON -DZLIB_ROOT="@TERMUX_PREFIX@" -DZLIB_LIBRARY="@TERMUX_PREFIX@/lib/libz.so" -DZLIB_INCLUDE_DIR="@TERMUX_PREFIX@/include"
 LLVM_CMAKE += -DLLVM_ENABLE_ZSTD=OFF
 ifeq ($(USE_POLLY_ACC),1)
 LLVM_CMAKE += -DPOLLY_ENABLE_GPGPU_CODEGEN=ON
